action/protocol/execution/testdata/datacopy.json

```diff
@@ -0,0 +1,28 @@
+{
+    "initBalances": [{
+        "account": "io1mflp9m6hcgm2qcghchsdqj3z3eccrnekx9p0ms",
+        "rawBalance": "1000000000000000000000000000"
+    }],
+    "deployments": [{
+        "rawByteCode": "608060405234801561001057600080fd5b50610347806100206000396000f3fe608060405234801561001057600080fd5b506004361061002b5760003560e01c8063574c807814610030575b600080fd5b61003861004e565b6040516100459190610231565b60405180910390f35b60606000600367ffffffffffffffff81111561006d5761006c6102d1565b5b6040519080825280601f01601f19166020018201604052801561009f5781602001600182028036833780820191505090505b509050601160f81b816000815181106100bb576100ba6102a2565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a905350602260f81b81600181518110610103576101026102a2565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a905350603360f81b8160028151811061014b5761014a6102a2565b5b60200101907effffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff1916908160001a90535060008151905060008167ffffffffffffffff81111561019c5761019b6102d1565b5b6040519080825280601f01601f1916602001820160405280156101ce5781602001600182028036833780820191505090505b509050600082602185018460208701600462010000fa9050826000602084013e8194505050505090565b600061020382610253565b61020d818561025e565b935061021d81856020860161026f565b61022681610300565b840191505092915050565b6000602082019050818103600083015261024b81846101f8565b905092915050565b600081519050919050565b600082825260208201905092915050565b60005b8381101561028d578082015181840152602081019050610272565b8381111561029c576000848401525b50505050565b7f4e487b7100000000000000000000000000000000000000000000000000000000600052603260045260246000fd5b7f4e487b7100000000000000000000000000000000000000000000000000000000600052604160045260246000fd5b6000601f19601f830116905091905056fea2646970667358221220ab496a0fa0ea6ff6e54fb6a9d37878e9e5361d6e3f640ab58d2ddee31465ede864736f6c63430008070033",
+        "rawPrivateKey": "cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1",
+        "rawAmount": "0",
+        "rawGasLimit": 5000000,
+        "rawGasPrice": "0",
+        "rawExpectedGasConsumed": 265114,
+        "expectedBalances": [],
+        "comment": "deploy datacopy contract"
+    }],
+    "executions": [{
+        "readOnly": true,
+        "rawPrivateKey": "cfa6ef757dee2e50351620dca002d32b9c090cfda55fb81f37f1d26b273743f1",
+        "rawByteCode": "574c8078",
+        "rawAmount": "0",
+        "rawGasLimit": 1000000,
+        "rawGasPrice": "0",
+        "rawExpectedGasConsumed": 12481,
+        "hasReturnValue": true,
+        "rawReturnValue": "000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000031122330000000000000000000000000000000000000000000000000000000000",
+        "comment": "the data of return is [0x11, 0x22, 0x33]"
```

Comment:
as we discussed, use fixed privatekey and timestamp, add the block hash/state root/receipt root values into test?